Sources/FormattingHelpers.swift

```diff
@@ -2029,6 +2062,12 @@ extension Formatter {
                 return declarationParser.index(of: .identifier("View"), after: someKeywordIndex) != nil
             }()
 
+            let isSwiftUIPropertyWrapper = mode == .visibility && { () -> Bool in
+                declarationParser.modifiersForDeclaration(at: declarationTypeTokenIndex, contains: { _, modifier in
+                    swiftUIPropertyWrappers.contains(modifier)
+                })
+            }()
```

Comment:
I don't think this immediately-executed closure is necessary since the body is just a single expression

```suggestion
            let isSwiftUIPropertyWrapper = mode == .visibility &&
                declarationParser.modifiersForDeclaration(at: declarationTypeTokenIndex, contains: { _, modifier in
                    swiftUIPropertyWrappers.contains(modifier)
                })
```